### ISSUE TITLE
customizing UI and add share screen entry in share content

### ISSUE THREAD

**Issue by omiddlk** [evilrat/flutter_zoom_sdk#37]
Hi, @evilrat 
I want to add share screen cell in share content action sheet but there was no tutorial for iOS SDK,
but in android there was this link:

https://marketplace.zoom.us/docs/sdk/native-sdks/android/mastering-zoom-sdk/in-meeting-function/customized-meeting-ui/share

is there any way to do with iOS zoom SDK?

**Comment by evilrat** [comment]
yes...it is possible on android......and i will look and provide solution on android too.....

**Comment by omiddlk** [comment]
@evilrat 
Thanks but I want to implement it in iOS as I said and don't know how
Any ideas?

**Comment by omiddlk** [comment]
@evilrat

**Comment by abdalmonem** [comment]
@omiddlk did you find any thing ?

**Comment by evilrat** [comment]
> @omiddlk did you find any thing ?

no still not
